tetris/impl/presets.py

```diff
@@ -0,0 +1,31 @@
+"""Collection of presets from various Tetris games."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from tetris import Engine
+
+if TYPE_CHECKING:
+    from tetris import BaseGame
+
+from tetris.impl import gravity
+from tetris.impl import queue
+from tetris.impl import rotation
+from tetris.impl import scorer
```

Comment:
```suggestion
from tetris.impl.gravity import InfinityGravity
from tetris.impl.queue import SevenBag
from tetris.impl.rotation import SRS
from tetris.impl.scorer import GuidelineScorer
```
the same applies here

Reply:
for this one I intentionally kept the scope larger so later on when more presets get added you don't have to individually specify each object